config/github/codeOwners

```diff
@@ -0,0 +1 @@
+* @mangs
```

Comment:
I don't think Babbel teams are available under the `babbel` org on GitHub, so I'll have to recruit some people and manually add them here individually

Reply:
You can sign me up if needed 👼